framework/Samplers/AdaptiveSobol.py

```diff
@@ -811,8 +809,8 @@ def _makeSubsetRom(self,subset):
     for inp in self.sorted:
       if self._checkCutPoint(subset,inp):
         #get the solution
-        inExisting = self.solns.getMatchingRealization(self._tupleToDict(inp))
-        soln = self._dictToTuple(inExisting['outputs'],output=True)
+        _,inExisting = self.solns.realization(matchDict=self._tupleToDict(inp))
```

Comment:
what is the meaning of "_,"?

Reply:
"_" is a Python-recognized empty variable, so that if a function returns multiple values, you can indicate that you don't care about certain values and not bother storing them locally.